CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_netsuite_source v0.10.0
+
+This is part of a batch release. 
```

Comment:
I don't think this is all that necessary and can probably be removed.
```suggestion
```

Reply:
Agreed.